packages/react-devtools-shared/src/devtools/views/Settings/GeneralSettings.js

```diff
@@ -86,6 +89,21 @@ export default function GeneralSettings(_: {}): React.Node {
         </div>
       )}
 
+      {isInternalFacebookBuild && (
+        <div className={styles.Setting}>
+          <label>
+            <input
+              type="checkbox"
+              checked={fbInternalFeaturesEnabled}
+              onChange={({currentTarget}) =>
+                setFbInternalFeaturesEnabled(currentTarget.checked)
+              }
+            />{' '}
+            Enable fb internal features
+          </label>
+        </div>
+      )}
```

Comment:
Instead of having a checkbox for everything, can we do this instead:
1. Make this PR contain only change to extension build script. Looks good, I think we can merge it
2. Based on `isInternalFacebookBuild`, display a text inside the settings panel, saying like "This is an internal build of React DevTools for Meta".

Then, for all upcoming features we will only add a checkbox if this is an internal feature that can't be generalized or if this feature is optional / expensive to use.